Nmap version detection (#1337)
Codecademy/docs: content/cybersecurity/concepts/nmap/terms/nmap-Version-Detection/nmap-Version-Detection.md

> ---

Reviewer: Can we change the filename from nmap-Version-Detection/nmap-Version-Detection.md to nmap-version-detection/nmap-version-detection.md?

Author: sure

Title: 'Nmap Version Detection'
Description: 'Version detection provides information on all identified open ports which interrogating the ports using probes that are understood by specific services.'
Subjects:
  - 'Computer Science'
  - 'Networking'
Tags:
  - 'Ethical Hacking'
  - 'Cybersecurity'
Catalog/content:
  - 'introduction-to-ethical-hacking'
  - 'paths/scan-systems-with-nmap'
---

> **Version detection** provides information on all identified open ports interrogating them using probes that generate specific responses by specific services. Version Detection is done with the `-sV` command and can identify the version number, the service type, the operating system, the hostname, etc…

Reviewer: ```suggestion
**Version detection** provides information on all identified open ports interrogating them using probes that generate specific responses by specific services. Version Detection is done with the `-sV` command and can identify the version number, the service type, the operating system, the hostname, etc.

```

> the command can be customized with the following options:

Reviewer: ```suggestion
The command can be customized with the following options:
```

    * --allports
    * --version-intensity
    * --version-light
    * --version-all
    * --version-trace

## Syntax

```shell
nmap -sV <localhost>
```

The command starts an assessment on <localhost>.

## Example

> The following paragraph demonstrates the use of Version Detection in _scanme.nmap.org_:

Reviewer: ```suggestion
The following example demonstrates the use of Version Detection using `scanme.nmap.org`:
```
